homeassistant/components/nws/weather.py

```diff
@@ -75,10 +74,10 @@ def convert_condition(
 
     if cond == "clear":
         if time == "day":
-            return ATTR_CONDITION_SUNNY, max(prec_probs)
+            return ATTR_CONDITION_SUNNY
         if time == "night":
-            return ATTR_CONDITION_CLEAR_NIGHT, max(prec_probs)
-    return cond, max(prec_probs)
+            return ATTR_CONDITION_CLEAR_NIGHT
```

Comment:
This line didn't have coverage before the change, either.

Reply:
Please add in a test to increase coverage

Reply:
@MatthewFlamm added test coverage